README.txt

```diff
@@ -1 +1,24 @@
-## This repo has the svg artwork for the Backed Community NFT and bunny traits for its avatar.
\ No newline at end of file
+## This repo has the svg artwork for the Backed Community NFT and bunny traits for its avatar.
+
+**How to Earn XP**
+
+**Earn Activity XP**
+
+- As a lender, provide your first loan (+1XP for every 5th)
+- As a borrower, pay interest on your first loan (+1XP for every 5th)
+- Approve your first lending strategy (+1XP for every 5th)
+- Lend or borrow the highest loan amount in a given month
+
+**Earn Contributor XP**
+
+- Leave your first constructive comment on Github (+1XP for every 5th)
+- Design assets or feature concepts used in production (+1XP for every 5th)
+- Contribute meaningful code on Github (+1XP for every 5th)
```

Comment:
I worry about the pressure words like "meaningful" put on us, would prefer to make this simpler/dumber.

Reply:
I think I would go for "merged PR (non-documentation)" or something, so it's clear what earns it. We can assume that codeowners will only approve "meaningful" code, so if we merge something it counts